### PR TITLE
Add library + conda env + tuto dir

### PR DESCRIPTION
**Pull request by apatlpo**

### CODE REVIEW

File: croco_osi/postp.py [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+        self.Np = self.N + 1
+
+        # Copy the horizontal dimenions to the hgrid object
+        self.hgrid.L  = self.L
+        self.hgrid.Lm = self.Lm
+        self.hgrid.M  = self.M
+        self.hgrid.Mm = self.Mm
+
+        # fills in actual value of f, f0 and beta0
+        f0 = 1.0313e-4
```
Comment (slgentil) [inline croco_osi/postp.py]: f0,beta should be in argument

File: croco_osi/postp.py [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+        # Extrapolate by one at each end (assumes equal grid spacing at the ends)
+        xxx=np.append(np.append(2*xx[0]-xx[1],xx),2*xx[-1]-xx[-2])
+        yyy=np.append(np.append(2*yy[0]-yy[1],yy),2*yy[-1]-yy[-2])
+
+        x_vert, y_vert = np.meshgrid(xxx,yyy)
+        self.hgrid = CGrid(x_vert, y_vert)
+
+        if check:
+            # Verify that CGrid class regenerated the right grid
+            a=np.max(np.abs(self.hgrid.x_u-self.his.variables['nav_lon_dom_U'][:]))
```
Comment (slgentil) [inline croco_osi/postp.py]: variables for navigation have changed: nav_lat_(rho/u/v/w), nav_lon_(rho/u/v/w)

File: doc/CONDA.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+Miniconda must be used with bash. If you want to use it with csh, add in your .cshrc (not ideal solution)
+```
+#
+#----------------------------------------------------------------
+# alias Miniconda
+#----------------------------------------------------------------
+#
+setenv PATH ${PATH}: /home/machine/username/miniconda3/bin
+alias source_activate 'setenv OLDPATH ${PATH};setenv PATH /home/machine/username/miniconda3/envs/\!*/bin:${PATH}'
```
Comment (slgentil) [inline doc/CONDA.md]: aliases no more useful with new versions of miniconda

File: croco_osi/postp.py (hunk 1 above)
Comment (apatlpo) [reply croco_osi/postp.py]: should be done, `_readgrid` may not be useful anymore however, see `_adjust_grid`

File: croco_osi/postp.py (hunk 2 above)
Comment (apatlpo) [reply croco_osi/postp.py]: this is now tretated in `_adjust_grid`

**Comment by apatlpo** [comment]
should be good to go

**Pull request merged**